### ISSUE TITLE
RSS: feed generation not working in Safari

### ISSUE THREAD

**Issue by frederikvoigt** [mediathekview/mediathekviewweb#103]
Hi,

there might be an issue with the feed generation in Safari for macOS.

When I click the button, macOS asks for the permission to open my RSS reader. The reader then opens, but it looks like it does not receive a valid feed URL. Javascript is enabled, and I tried it with all content and tracking blockers completely disabled. What else can I try? It works like a charm with Chrome though.

Such a great service, by the way!

**Comment by bagbag** [comment]
I have no Idea why that could be and unfortunately I cannot test this as I dont have an Apple device.
Does the reader shows the URL which it receives?

**Comment by frederikvoigt** [comment]
No. It opens, but does not even show the dialogue to confirm the addition—which probably is a sign that it does not receive anything.

**Comment by bagbag** [comment]
As I have no Idea whats the reason for, nor how to test and fix that (I mean all MVW is doing here is opening a link), I think I cannot help you.

**Comment by frederikvoigt** [comment]
Thanks for the feedback! No need to bother the forum members, I can just use Chrome. Yep, that's less comfortable, but mainly I wanted to let you know there's a bug. ;) Best regards!
